github-dark.css

```diff
@@ -5250,3 +5250,94 @@
   }
   /* Literal.Number.Integer.Long */
 }
+
+@-moz-document domain("github.com") {
+  /* ui when there are no workflows setup */
+  .deployments-timeline-item + div:not(.deployments-timeline-item) {
+    background-image: linear-gradient( rgba(28, 28, 28, 0), rgba(28, 28, 28, 1)) !important;
+  }
+  [style*="background-color: #e1e4e8"] {
+    background-color: #8e8e8e !important;
+  }
+  [style*="color: #e1e4e8"] {
+    color: #8e8e8e !important;
+  }
+  /**
+   * this is the same form as when you're committing from the site
+   * this means there's a small arrow on the left (which points at
+   * the user's avatar) and this ends up showing in the dropdown too
+   */
+  .file-commit-form .dropdown-menu .commit-form:after {
+    content: none !important;
+  }
+  .file-commit-form .commit-form {
+    background: #181818 !important;
+  }
+}
+
+@-moz-document domain("launch-editor.github.com") {
+  body {
+    background: #181818 !important;
+  }
+  .editor-node[style*="background-color: rgb(68, 77, 85)"] {
+    background: #343434 !important;
+  }
+  .editor-node {
+    background: #1c1c1c !important;
+  }
+  .blankslate-box {
+    background: #242424 !important;
+  }
+  .overlay {
+    background: rgba(27, 31, 35, .5) !important;
+  }
+  .blankslate-box, .editor-node, .editor-node > div {
+    border-color: #484848 !important;
```

Comment:
I'm not sure I prefer the general approach of having styles like this that could be auto-generated here. Did you check if `grunt generate` already covers these selectors? If so, you can remove them here.

Reply:
These styles come from a different stylesheet than what we auto-generate from. The current url for this is https://launch-editor-assets.githubusercontent.com/_next/static/css/commons.13e866fa.chunk.css

Reply:
Hmm, is there a public HTML page that includes a link to that style?

Reply:
If I load the base url `https://launch-editor.github.com/render` I get an error page (when logged in) and that has the stylesheet in it. I'm not sure what will show for a user without access though.